### ISSUE TITLE
research data lesson design: can 23RD things be broken up into a LC stand alone lesson?

### ISSUE THREAD

**Issue by jt14den** [LibraryCarpentry/23-rd-things#1]
@pitviper6 @U2NG @libcce and others Can we remix this into a RD lesson for LC? We can look at the lesson design episode: https://carpentries.github.io/instructor-training/14-lesson-study/ and Greg's: http://third-bit.com/teaching/lessons.html.

**Comment by U2NG** [comment]
Good idea, I think there is enough 23rd things content to dev a RD lesson for LC.

**Comment by ccronje** [comment]
Hi @jt14den and folks! Can I join you on this one? I was wondering if we can incorporate parts of this FAIR data module in here http://www.reproducibleimaging.org/module-FAIR-data/ What do you think?

**Comment by jt14den** [comment]
@ccronje @U2NG Would it be easier to start fresh? With a lc-research-data in with a new template and then figure out what can be brought in from 23RD and the FAIR data module?

**Comment by ccronje** [comment]
@jt14den @U2NG yeah that might be easier. What's a good way to plan a new lesson? Was thinking maybe a google doc?

**Comment by jt14den** [comment]
Google doc is good with me. You can invite timdennis@g.ucla.edu

**Comment by ccronje** [comment]
Or an etherpad? @weaverbel can you please create a SWC etherpad for lc-research-data?

**Comment by ccronje** [comment]
Planning over here folks http://pad.software-carpentry.org/lc-research-data

**Comment by fionajones** [comment]
Hi all! Hi @ccronje :)
I’ll take a look at this too because I’m interested and it seems very relevant at MPOW :)

**Comment by ccronje** [comment]
Hi @fionajones that would be awesome!

**Comment by jt14den** [comment]
I created a new repo for [lc-research-data](https://github.com/LibraryCarpentry/lc-research-data) - blank slate, so we can put from the lessons into here.

**Comment by ccronje** [comment]
Thanks @jt14den what would you like us to work on today? I'm not sure I have my head around reverse instructional design so please lead the way :-)

**Comment by jt14den** [comment]
Sorry @ccronje  @U2NG @fionajones, I ate too much mozcake, but also had fun helping new folks with lessons and git at the sprint, so I didn't do much with the reverse instructional design process. @pitviper6 and @libcce did have some thoughts about a way forward. I suggest we close off work on this repo and move to the  https://github.com/LibraryCarpentry/lc-research-data and continue the work. I've added a `design.md` file to that repo, that I hope we can populate with work from the [etherpad](http://pad.software-carpentry.org/lc-research-data).
